### PR TITLE
Use predefined BMH status in examples (master)

### PR DESCRIPTION
**Pull request by wgslr**
Port changes from #212 to branch master.

### CODE REVIEW

File: docs/dev-setup.md [hunk 1]
```diff
@@ -331,11 +331,16 @@ controller is doing. You can also proceed to create/update/delete
 ### Deploy an example cluster
 
 ```sh
-    make deploy-examples
+make deploy-examples
 ```
 
+If you want to simulate the behavior of the metal3 operator, change
```
Comment (fmuyassarov) [inline docs/dev-setup.md:337]: nit,
```suggestion
If you want to simulate the behavior of the metal3 baremetal operator, change
```

**Review by fmuyassarov — APPROVED** [review]

File: docs/dev-setup.md (hunk 1 above)
Comment (fmuyassarov) [reply docs/dev-setup.md:337]: I can lgtm once this is fixed

**Review by furkatgofurov7 — COMMENTED** [review]
/approve

**Pull request merged**